lib/splash.js

```diff
@@ -39,7 +41,13 @@ function logKey(key) {
 }
 
 function logValue(key, value) {
-    return "| " + indent() + " " + key + ": " + value + "\n";
+    var hideValue = _.reduce(KEYS_TO_HIDE, function(memo, keyToHide) {
+        return memo || key.indexOf(keyToHide) >= 0;
```

Comment:
Wait... The .indexOf() is done on the `key`.
I think at this stage, the `key` should not be an array...